---
kP : 3
kI : 0
kD : 0
kP_r : 3
kI_r : 0
kD_r : 0
max_linear_speed : 0.1
min_linear_speed : 0.0001
max_angular_speed : 0.1
min_angular_speed : 0.0001
linear_tolerance : 0.1 # 2.5cm
angular_tolerance : 0.053   #3/180*pi' # 3 degrees
forward_movement_only : True